### PR TITLE
PR of all updates since November 2022

### PR DESCRIPTION
**Pull request by opoudjis**
### Metanorma PR checklist

 - [ ] Breaking changes (list related PRs)
 - [ ] Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] Gem with native library introduced

### CODE REVIEW

**Comment by ronaldtse** [comment]
@kwkwan please take note of the latest changes from @opoudjis .

**Comment by ronaldtse** [comment]
@kwkwan can you please fix the link checker job here and merge the changes? Thanks.

File: _layouts/tutorial.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  items:
+    - title: Metanorma tutorial for beginners
+      path: /
+
+    - title: Introduction to Metanorma
+      items:
+        - title:
+          path: /tutorials/template/
+        - title:
+          path: /tutorials/template/
```
Comment (ronaldtse) [inline _layouts/tutorial.html:15]: @kwkwan can you please help fill these in?

**Review by ronaldtse — CHANGES_REQUESTED** [review]
For the navigation links that are missing, let's comment them out first, and merge this.

**Comment by ronaldtse** [comment]
Frankly I am not happy with the many "merges" across branches of this PR. 

But it's going to take too much time to rebase cleanly. Merging now.

**Comment by opoudjis** [comment]
Sound version management practice presupposes that 40 PRs do not go unactioned for over a year. Hopefully this does not happen again.

**Pull request merged**